index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Holliday Planner</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+    <div class="navbar">
+        <img src="./picture/Logo.png" alt="brand-logo" width="80" height="auto">
```

Comment:
separation of concerns: height/width are referring to presentation, so these should be handled in css